nerfstudio/models/nerfacto.py

```diff
@@ -271,7 +271,7 @@ def bias(x, b):
     def get_outputs(self, ray_bundle: RayBundle):
         ray_samples: RaySamples
         ray_samples, weights_list, ray_samples_list = self.proposal_sampler(ray_bundle, density_fns=self.density_fns)
-        field_outputs = self.field(ray_samples, compute_normals=self.config.predict_normals)
+        field_outputs = self.field.forward(ray_samples, compute_normals=self.config.predict_normals)
```

Comment:
The output type doesn't get resolved probably when we call `self.field()` directly, since the annotations live under `forward` and `__call__` is effectively annotated with `*args` / `**kwargs`.  This was causing some issues in conjunction with the previously incorrect `scale_gradients_by_distance_squared` annotation.

With the gradient scaling signature fixed this change is no longer needed, do you think there are good reasons for removing it? I guess hooks?